pkg/writer/writer.go

```diff
@@ -13,21 +13,28 @@ import (
 )
 
 type Writer struct {
-	Indent int
-	Format formats.Format
+	RenderOptions   map[string]*native.RenderOptions
+	SerialzeOptions map[string]*native.SerializeOptions
+	Format          formats.Format
 }
 
 var (
-	regMtx        sync.RWMutex
-	serializers   = make(map[formats.Format]native.Serializer)
-	defaultIdent  = 4
-	defaultFormat = formats.CDX15JSON
+	regMtx               sync.RWMutex
+	serializers          = make(map[formats.Format]native.Serializer)
+	defaultRenderOptions = &native.RenderOptions{
+		CommonRenderOptions: native.CommonRenderOptions{
+			Indent: 4,
+		},
+	}
+	defaultSerializeOptions = &native.SerializeOptions{}
+	defaultFormat           = formats.CDX15JSON
```

Comment:
I don't have a strong opinion here but perhaps we should seek consensus on moving the default to CDX 1.5. At least check with the cohort to see if there are any objections.

Reply:
Probably best to use our "most complete" serializer as the default choice.  That's unlikely to be CDX 1.5 at the moment.  I've been working on SPDX 2.3 mostly and that's looking pretty good.  I haven't looked at the CDX 1.5 support at all yet to see how complete our support is.